addons/hr/models/hr.py

```diff
@@ -186,6 +188,7 @@ def _default_image(self):
         'res.partner', 'Work Address')
     work_phone = fields.Char('Work Phone')
     mobile_phone = fields.Char('Work Mobile')
+    phone = fields.Char(related='address_home_id.phone', related_sudo=False, string="Private Phone", groups="hr.group_hr_user")
```

Comment:
how about no?

Reply:
"no" as the field name ?

Reply:
No !

cc @tde-banana-odoo

Reply:
no as "don't do drugs or other illegal operations that you don't to have your name on it when we blame the source code"